packages/frontend/src/features/semanticViewer/components/ContentResults.tsx

```diff
@@ -53,6 +60,50 @@ const ContentResults: FC = () => {
         fields,
     ]);
 
+    const thSortConfig = useMemo(() => {
```

Comment:
this duplication also makes me think weather this should live inside components and not runner or a model

Reply:
That's a great point, I don't like this very much either, but the reason why I did this was so I could pass a `onClick` handler to take care of mutating the sort state, this calls a dispatch so it isn't really doable in the runner 🤔 maybe we can figure out another solution?

Reply:
Yep, we can talk about it, but I think this PR is ready to be merged. I wouldn't block a PR for this.